### PR TITLE
Add 12.8.1 metadata + update metadata process

### PR DESCRIPTION
**Pull request by tas50**
One last doc update, use the latest gems for generating the metadata, add 12.8.1

### CODE REVIEW

File: chef_dsl_metadata/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Generating metadata for a specific chef version
 - run 'bundle install' to install appraisal
 - update Appraisals file with new chef version(s)
-- run 'appraisal chef_12.7.2 bundle install' to generate the Gemfile and Gemfile.lock
-- run 'appraisal chef_12.7.2 rake generate_chef_metadata' to generate json for that version of chef
+- run 'bundle exec appraisal install'. It will fail, but it will still generate the Gemfile
```
Comment (lamont-granquist) [inline chef_dsl_metadata/README.md]: missing the version on this line or does it no longer work?

not sure why we need to bother bumping to appraisal 2.x either does it actually buy us anything?

**Comment by lamont-granquist** [comment]
:+1: either way

File: chef_dsl_metadata/README.md (hunk 1 above)
Comment (tas50) [inline chef_dsl_metadata/README.md]: The bundle install command fails as the gem isn't there yet.  The appraisal install command generates all the Gemfiles.

```
appraisal chef_12.8.1 bundle install
WARN: Unresolved specs during Gem::Specification.reset:
      rake (>= 0)
WARN: Clearing out unresolved specs.
Please report a bug if this causes problems.
>> BUNDLE_GEMFILE=/Users/tsmith/dev/oss/foodcritic_mine/chef_dsl_metadata/gemfiles/chef_12.8.1.gemfile bundle install
/Users/tsmith/dev/oss/foodcritic_mine/chef_dsl_metadata/gemfiles/chef_12.8.1.gemfile not found
```

Comment (tas50) [inline chef_dsl_metadata/README.md]: The appraisal bump doesn't actually get us anything. We did a similar bump about a year ago to just bring it up to current.  It's just a pile of random bugfixes and some new features we don't use.

**Pull request merged**